jerry-core/jerry-port.h

```diff
@@ -139,6 +140,14 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+
+typedef uint32_t (*jerry_job_handler_t) (void *);
+
+void jerry_port_jobqueue_enqueue (jerry_job_handler_t handler, void *job_p);
```

Comment:
Yup, you're right. I've thought about this a bit more and I'm starting to doubt whether my comment here makes sense. The source of the ES6 job queue events/jobs is from things internal to JerryScript (for now only Promise), while for something like `setTimeout()`, the event source is the native/host system, which would put an event into the (existing) native event loop.